src/test/java/org/alfresco/repo/security/authority/AuthorityServiceTest.java

```diff
@@ -351,9 +365,14 @@ public void testZones()
     @Category(RedundantTests.class)
     public void test_ETWOTWO_400()

```

Comment:
I think this test needs to be deleted as it is essentially the same as testCreateAuthorityWithIllegalCharacters now.

Reply:
Removed the test.